public/js/weatherapp.js

```diff
@@ -1,26 +1,23 @@
-console.log('client side js is loaded!')
-
-
 const weatherForm = document.querySelector('form')
 const search= document.querySelector('input')
-const msg1 = document.querySelector('#msg1')
-const msg2 = document.querySelector('#msg2')
+const weatherSearchTitle = document.getElementById('weather-search-title')
+const weatherSearchResults = document.getElementById('weather-search-results')
 
-weatherForm.addEventListener('submit', (e) => {
+weatherForm.addEventListener('submit', e => {
```

Comment:
when should we write (e) and when should we write e? if we have a e.target.value that we have to store in a variable, then we need to write (e), but if we only use a function on itself, then we don't need the ()? Can you explain more on that? I think I vaguely remember if you have one parameter in your argument, you can omit (), but I'm still not clear ...

Reply:
Anytime that you have only a single argument then the `()` are optional. I generally remove them since I think it looks cleaner without them, but you can leave them in and it will still work just fine. There is no difference between the two other than aesthetics.